packages/dapp-svelte-wallet/ui/src/main.js

```diff
@@ -4,7 +4,7 @@ import App from './App.svelte';
 import AppV2 from './AppV2.svelte';
 
 const urlParams = new URLSearchParams(window.location.search);
-const isV2 = urlParams.get('ui') === 'v2';
+const isV2 = urlParams.get('ui') !== 'v1';
```

Comment:
I'm okay with making it the default as long as you agree it's in a ready enough state for all users. We can always manually add v2 for the demo, I was just hesitant about enabling it for all users in an unfinished state. Not that it's not already an improvement over v1, but I didn't want people to look at it expecting it to be fully done and wonder why there's still a lot of rough edges.

Reply:
> Not that it's not already an improvement over v1, but I didn't want people to look at it expecting it to be fully done and wonder why there's still a lot of rough edges.

Fair enough.  It's reasonable to add the `?ui=v2`, so I put that in the demo script doc.